rfps/project_discovery_data_insights_rfp.md

```diff
@@ -0,0 +1,84 @@
+# Drips Widget - Proof of Concept
+
+- **Status:** Open 
+- **Proposer:** [@bordumb](https://github.com/bordumb)
+- **Your Project(s):** [optional]: https://github.com/radicle-dev
+- **Projects you think this work could be useful for** [optional]: everything on Radicle
+- **Teams/People that could deliver the RFP** [optional]: Looking...
+
+## Project Description :page_facing_up:
+
+Create a way to rank, categorize, and surface relationships on the Radicle network.
+
+The vision for this product is **BIG**.
+
+We could see it being used by any sort of creator, including but not limited to:
+* Ranking top Orgs by number of contributors, number of Drips outgoing/incoming, etc.
+* Ranking top open source contributors by number of Orgs, number of contributions, or Drips received 
+* Recommending work to contributors that is similar (e.g. use collaborative filtering based on commit/contribution data)
+* Recommending work to Drip to (e.g. use collaborative filtering based on Drips data)
+
+> The main goal here is surface insights that **make navigation and discovery within the Radicle ecosystem easier**.
+
+## Deliverables :nut_and_bolt:
+
+- **Total Estimated Duration:** up to 4 months (negotiable; please see `Milestone` sections)
+- **Full-time equivalent (FTE):** 2
+- **Total Costs:** roughly 72,000 USDC
+
+### Milestone 1
+
+- **Estimated Duration:** 1 month
+- **FTE:** 1
+- **Costs:** 12,000 USDC
+
+| Number | Deliverable  | Specification                                                                                                                              |
+|--------|--------------|--------------------------------------------------------------------------------------------------------------------------------------------|
+| 1.     | Backend code | Queries that do basic ranking of Orgs by Drips activity (e.g. ranking by # of Drips ingoing/outgoing, amount of Drips, etc.)               |
+| 2.     | Backend code | Queries that do basic ranking of Orgs by contribution activity (e.g. most recent contributions, most unique contributors, etc.)            |
+| 3.     | Backend code | Queries that do basic ranking of Communities by Drips activity (e.g. top orgs by # of Drips ingoing/outgoing, amount of Drips, etc.)       |
+| 5.     | Backend code | Queries that do basic ranking of Contributors by development activity (e.g. most Orgs joined, most commits, etc.)                          |
+| 6.     | Backend code | Queries that do basic ranking of Contributors by development activity (e.g. ranking by # of Drips ingoing/outgoing, amount of Drips, etc.) |
```

Comment:
It's still early but I see also ability to allow workstreams to be a part of this (eventually in a later milestone) to see
- Org with most open / valued / active / etc. RFPs
- Contributors which are most active in workstreams, best valued, etc.

Reply:
> Org with most open / valued / active / etc. RFPs

This is a super important one!!! 
Can't believe I forgot it.

The main framework I want this to address is:
- Supply side of labor (good discovery to help contributors)
- Demand side of labor (good discovery to help DAOs/Orgs/Communities)

These fit perfectly.

Reply:
Resolving this as I've broken things out a bit more